include/hipSYCL/sycl/libkernel/group.hpp

```diff
@@ -216,12 +244,43 @@ struct group
   }
 
   HIPSYCL_KERNEL_TARGET
-  size_t get_linear() const
+  size_t get_group_linear_id() const
   {
     return detail::linear_id<Dimensions>::get(get_id(),
                                               get_group_range());
   }
 
+  [[deprecated("Use get_group_linear_id() instead.")]]
+  HIPSYCL_KERNEL_TARGET
+  size_t get_linear() const
+  {
+    return get_group_linear_id();
+  }
+
+#ifdef SYCL_DEVICE_ONLY
+  id_type get_local_id() const
+  {
+    return detail::get_local_id<Dimensions>();
+  }
+
+  size_t get_local_id(int dimension) const
+  {
+    return detail::get_local_id<Dimensions>(dimension);
+  }
+#endif
+
+  range<Dimensions> get_max_local_range() const{
```

Comment:
I think those functions should be marked `HIPSYCL_KERNEL_TARGET` for consistency, no?